client/index.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-undef */
+
+// const { default: axios } = require('axios')
+
+/* ---------------------------- */
+// REMEMBER TO FLIP THIS WHEN NOT USING LIVE SERVER
+
+const BASE_URL = 'http://localhost:3001/api'
+// const BASE_URL = '/api'
+
+/* ---------------------------- */
+
+const entriesSection = document.querySelector('.entries-container')
+const infoSection = document.querySelector('h6')
+
+const entriesArray = []
```

Comment:
Maybe Contacts array for a more meaningful name. And Contact Instead of entry in the next lines